Sources/NIOHTTP2/HTTP2StreamMultiplexer.swift

```diff
@@ -30,13 +30,15 @@ public final class HTTP2StreamMultiplexer: ChannelInboundHandler, ChannelOutboun
     private var streams: [HTTP2StreamID: HTTP2StreamChannel] = [:]
     private let inboundStreamStateInitializer: ((Channel, HTTP2StreamID) -> EventLoopFuture<Void>)?
     private let channel: Channel
+    private var context: ChannelHandlerContext!
```

Comment:
needs to be nilled out in `handlerRemoved`

Reply:
Addressed.